webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -665,11 +665,13 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
 
   cancelRubric() {
 
-    const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
+    const rubricGradingElem = document.getElementsByTagName("sakai-rubric-grading");
+    if (rubricGradingElem.length <= 0) {
+      return;
+    }
+    const rubricGrading = rubricGradingElem.item(0);
     rubricGrading && rubricGrading.cancel();
-
-    this.rubricShowing = true;
-    this.doneWithRubricDialog();
+    this.rubricShowing && this.doneWithRubricDialog();
```

Comment:
Are you sure this is what you intended? This isn't the same as the two statements you've replaced.  It seems to make sense ("If the rubric is showing at the moment, close the rubric dialog")

Reply:
My intention is to not set rubricShowing when we are canceling. I think this should be setted when toggle method is executed to show the rubric... I tested the grader after my changes and I think it's fine. Also, any suggestion is always positive.